app/services/dqt/v1/teacher.rb

```diff
@@ -0,0 +1,25 @@
+module Dqt
+  module V1
```

Comment:
we discussed this with the TRA team and unfortunately V3 is missing some features so we had to go back to V1 for now. Since this will all be deprecated soon they agreed for us to go with V1

Reply:
Ah yes, we had a meeting about this and found v3 doesn't include some things we need. We're sticking with v1 for migration, and likely replace all this in the future.